### PR TITLE
Split clipboard data by any whitespace characters and commas

### PR DESCRIPTION
**Pull request by johnste**

### CODE REVIEW

**Comment by johnste** [comment]
Please let me know if this doesn't make sense. Maybe the regex could be configurable? This would support my use case (posting multiple lines of data).

File: lib/ReactTags.js [hunk 1]
```diff
@@ -190,7 +190,7 @@ var ReactTags = React.createClass({
     e.preventDefault()
     const clipboardData = e.clipboardData || window.clipboardData
     const string = clipboardData.getData('text')
-    string.split(/[ ,]+/).forEach((tag) => this.props.handleAddition(tag))
+    string.split(/[\s,]+/).forEach((tag) => this.props.handleAddition(tag))
```
Comment (prakhar1989) [inline lib/ReactTags.js]: Can you add a comment on what this regex supposed to do? Will help developers save a Google search :)

File: test/reactTags-test.js [hunk 2]
```diff
@@ -56,6 +56,29 @@ describe("ReactTags", () => {
 
   });
 
+  it("handles the paste event and splits the clipboard on whitespace or comma characters", () => {
```
Comment (prakhar1989) [inline test/reactTags-test.js]: Tests looks sweet! 🍬  Thanks for adding the tests 🍻

**Comment by prakhar1989** [comment]
Heya @johnste - This looks great! Thanks for sending this PR! I think this is a reasonable default regex. If more users want this to be configurable we can do that later on. For now, I added just one comment for you to address following which I can merge and ship this change. 

Thanks again for sending this over 👍

File: lib/ReactTags.js (hunk 1 above)
Comment (johnste) [inline lib/ReactTags.js]: Sure!

**Comment by prakhar1989** [comment]
@johnste : Merged and published your changes! Thanks again for contributing 👯

**Comment by johnste** [comment]
My pleasure - thanks for a good react component.

**Pull request merged**